woodwork/statistics_utils.py

```diff
@@ -306,3 +342,81 @@ def _get_value_counts(dataframe, ascending=False, top_n=10, dropna=False):
         values = list(df.to_dict(orient="index").values())
         val_counts[col] = values
     return val_counts
+
+
+def _get_numeric_value_counts_in_range(series, _range):
+    """Count the number of occurrences of integers present in a series with values defined
+    by a range of integers. Null values will be ignored.
+
+    Args:
+        series (pd.Series): data from which to determine the number of occurrences of values
+        _range (type(range)): sequence of integers defining the values for which counts should be made
+
+    Returns:
+        value_counts (list(dict)): a list of dictionaries with keys `value` and
+            `count`. Output is sorted in ascending order by the values defined in
+            the input range.
+    """
+    frequencies = series.value_counts(dropna=True)
+    return [
+        {"value": i, "count": frequencies[i] if i in frequencies else 0} for i in _range
+    ]
+
+
+def _get_top_values_categorical(series, num_x):
+    """Get the most frequent values in a pandas Series. Will exclude null values.
+
+    Args:
+        column (pd.Series): data to use find most frequent values
+        num_x (int): the number of top values to retrieve
+
+    Returns:
+        top_list (list(dict)): a list of dictionary with keys `value` and `count`.
+            Output is sorted in ascending order based on the value counts.
+    """
+    frequencies = series.value_counts(dropna=True)
+    df = frequencies.head(num_x).reset_index()
+    df.columns = ["value", "count"]
+    top_lt = list(df.to_dict(orient="index").values())
+    top_lt = sorted(top_lt, key=lambda i: (i["count"], i["value"]))
+    return top_lt
+
+
+def _get_oldest_value_counts(column, num_x):
+    """Get the the number of occurrences of the x oldest values in a datetime column.
+
+    Args:
+        column (pd.Series): data to use find value counts
+        num_x (int): the number of values to retrieve
+
+    Returns:
+        value_counts (list(dict)): a list of dictionary with keys `value` and
+            `count`. Output is sorted in ascending order by date.
+    """
+    datetimes = pd.to_datetime(column, infer_datetime_format=True, errors="coerce")
```

Comment:
The column will already be a datetime column, right? So no need to re-cast it? 
```suggestion
    datetimes = column
```

Reply:
Yeah, we can remove that `pd.to_datetime` col since Woodwork would have already done that conversion. I had to rework the test a little bit because of this though to remove the string values which would never be allowed in a Woodwork datetime column, but overall this cleans things up.